kustomize/data_source_kustomization_build.go

```diff
@@ -17,6 +18,15 @@ func dataSourceKustomization() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"load_restrictor": &schema.Schema{
```

Comment:
In the schema, even if for now we only implement the loadrestrictions, I think we should still wrap it in a hash map. Because I'm sure somebody will eventually have a requirement for another one of the [supported options](https://github.com/kubernetes-sigs/kustomize/blob/e4bbd04a43dac9e566fb8054efdcdcfe122f312f/api/krusty/options.go#L39).

And I think it would be cleaner to not add all of them to the schema at the top level? What do you think about that @darren-reddick?

Reply:
In fact there [was already a question](https://github.com/kbst/terraform-provider-kustomization/issues/64) about making the plugins configurable, which I suppose could also go into that hash map. I'm not saying you need to implement that as well. Just saying, wrap the load_restrictions into a hash map, and maybe it will be extended in the future.

Reply:
OK. Makes sense - just re-reading the issue and see that you hinted at this. I'll update this.